applications/IgaApplication/custom_python/iga_python_application.cpp

```diff
@@ -36,6 +36,8 @@ PYBIND11_MODULE(KratosIgaApplication, m)
     ;
 
     AddCustomUtilitiesToPython(m);
+
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, NURBS_CONTROLPOINT_WEIGHT)
```

Comment:
CONTROL_POINT?

Reply:
We have the same discussion in another branch. My idea was `Nurbs.ControlPoint.Weight`. It's a matter of taste and I will probably change it with the other branch.

Reply:
Just asking